test/tests.spec.ts

```diff
@@ -12,6 +12,7 @@ import assert = require("assert");
 import { FAKE_CERT, TEST_CERT } from "./types";
 import { signXmlResponse } from "../src/utility";
 import { parseDomFromString, xpath } from "../src/xml";
+import { Console } from "console";
```

Comment:
Why is this line needed?

Reply:
> Why is this line needed?

I's a mistake, I beg you pardon for this. I'm so sorry to be in hurry and let this commit out. Yesterday I spent my time wondering why I was unable to run ci & test on the codebase. So I end up installing nvm and 14.x version of node in WSL on windows. My previous working environment was wrong. Node 16.x, windows and I got wrong results from test run. The same code base on the new environment breaks up only 20 tests instead of 25. I don't know why and will take a look on this too, but after fixing my errors and completing my task. This long answer is menta to be a good response to your question. But to be onest, the right one should be "poor man debugging".